server/routes/email.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+const { sendEmail } = require("../controllers/email");
+
+// @route post /email/send
+router.route("/send").post(sendEmail);
+
+module.exports = router;
```

Comment:
We probably don't need to have a route to send an email, instead, in our controllers we will do logic to send emails as opposed to hitting a route.